CHANGELOG.md

```diff
@@ -26,7 +26,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
   * `cortex_ingester_queried_ephemeral_series`
 * [FEATURE] Ruler: added `keep_firing_for` support to alerting rules. #4099
 * [FEATURE] Distributor, ingester: ingestion of native histograms. The new per-tenant limit `-ingester.native-histograms-ingestion-enabled` controls whether native histograms are stored or ignored. #4159
-* [FEATURE] Query-frontend: Introduce experimental `-query-frontend.query-sharding-target-series-per-shard` to allow query sharding to take into account cardinality of similar requests executed previously. #4121 #4177 #4188
+* [FEATURE] Query-frontend: Introduce experimental `-query-frontend.query-sharding-target-series-per-shard` to allow query sharding to take into account cardinality of similar requests executed previously. #4121 #4177 #4188 #4254
```

Comment:
Should we mention that this feature uses results-cache, even if results caching is disabled?

Reply:
Good idea.

Reply:
Absolutely, does https://github.com/grafana/mimir/pull/4254/commits/b380a1af9b3e02326008c92271003a26f274b4e8 sound sensible?

Reply:
I think Peter intended to mention it in the CHANGELOG entry too.

Reply:
like so: https://github.com/grafana/mimir/pull/4254/commits/3578a14502a9bc71718a01ae17dbc0149950b9b5?

Reply:
That looks good to me, thanks!